app/src/features/settings/components/Profile/ManageTeams/TeamViewer/MembersDetails/AddMemberModal/index.jsx

```diff
@@ -284,13 +284,7 @@ const AddMemberModal = ({ isOpen, toggleModal, callback, teamId: currentTeamId,
                             isLoggedInUserAdmin={isLoggedInUserAdmin}
                             loggedInUserId={loggedInUserId}
                             handleMemberRoleChange={(_, updatedRole) => {
-                              let role = updatedRole;
-
-                              if (role === "user") {
-                                role = WorkspaceMemberRole.write;
-                              }
-
-                              setUserInviteRole(role);
+                              setUserInviteRole(updatedRole);
```

Comment:
if InviteRole now is of type TeamRole NOT WorkspaceRole

Can we update the [default setState](https://github.com/requestly/requestly/blob/42cc9bf7f294877d259f2996e8bf99f38db4b534/app/src/features/settings/components/Profile/ManageTeams/TeamViewer/MembersDetails/AddMemberModal/index.jsx#L32) to use TeamRole instead of WorkspaceRole and similar other corresponding changes in this file?

Reply:
Yes